.github/workflows/build_package.yml

```diff
@@ -127,7 +123,79 @@ jobs:
               -DIREE_BUILD_PYTHON_BINDINGS=ON \
               -DIREE_BUILD_SAMPLES=OFF && \
             python ./main_checkout/build_tools/cmake/cmake_ci.py \
-              --build ./iree-build --target install"
+              --build ./iree-build --target install-IreePythonPackage-rt-stripped"
+          # TODO: cibuildwheel sanity checks this, but our setup.py is the
+          # *output* of the build :( File a bug.
```

Comment:
File a bug against IREE or a different project? (this is copy-pasted a few times)

Reply:
https://github.com/joerick/cibuildwheel/issues/515

Basically, WAI